### PR TITLE
Updates resources for researchers per Olga's request

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2897](https://openedx.atlassian.net/browse/DOC-2897)

Adds suggested changes per Olga's request.
### Reviewers
- [x] Subject matter expert: @stroilova 
- [x] Doc team review (sanity check): @srpearce @pdesjardins @catong 
- [ ] Product review: @shnayder 

FYI: @nedbat 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add to edx-analytics-dashboard
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -230,13 +240,22 @@ version, select **v: latest** at the lower right of the page, and then select
...
-======
-Email
-======
+==============================
+Discussion Forums and Email
+==============================
+
+Researchers, edX data czars, and members of the global edX data and analytics
+community can  post and discuss questions in our public research forum, the
+`openedx-analytics`_ Google group.
```
Comment (stroilova) [inline en_us/shared/preface.rst:250]: as discussed offline, perhaps clarify by changing the comma to a colon? Or just say e.g. "in openedx-analytics(link) - our public research form"

**Comment by stroilova** [comment]
👍  ( with a minor clarification question on the public forum section, inline )

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -211,15 +211,25 @@ monitored by edX staff.
...
+collecting data about courses running on their sites and conducting research
+projects.
+
 ==============
 Documentation
 ==============
 
-The `edX Research Guide`_ is available on the docs.edx.org web page.
+The `edX Research Guide`_ is available on the docs.edx.org web page. While
+written primarily for data czars and researchers at partner institutions, this
```
Comment (catong) [inline en_us/shared/preface.rst]: "While written" -> "Although it is written"

File: en_us/shared/preface.rst [hunk 3]
```diff
@@ -211,15 +211,25 @@ monitored by edX staff.
...
+guide can also be a useful reference for members of the Open edX community.
 
 This guide opens in your web browser, with a **Search docs** field and links to
```
Comment (catong) [inline en_us/shared/preface.rst]: "This guide" followed by "that guide" is confusing. Clarify what "that guide" refers to?

**Comment by catong** [comment]
:+1: LGTM with a minor edit and one question to clarify something not actually changed in this PR.

**Pull request merged**